### ISSUE TITLE
Errors when using rm_pool

### ISSUE THREAD

**Issue by Licenser** [epgsql/pooler#26]
When using rm_pool to remove a pool the process ends with an error given that rm_pool is not a crash behavior this probably should not be treated as an error?

Example:

```
2014-07-30 17:08:52.708 [error] <0.255.0> Supervisor 'pooler_howl@localhost:4240_member_sup' had child mdns_client_lib_worker started with {mdns_client_lib_worker,start_link,undefined} at <0.259.0> exit with reason killed in context child_terminated
```

**Comment by sata** [comment]
Isn't this because of Lager interpreting Supervisor Reports as errors?[1]

[1] https://github.com/basho/lager/issues/153

**Comment by sata** [comment]
#66 resolves this. It's still going to brutally kill the child in the same way but it won't generate a supervisor report since it's the supervisor itself who terminates the child.

**Comment by seriyps** [comment]
I believe it is solved by #66, so, closing now. Feel free to reopen if disagree.
